flathub.json

```
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["i386", "x86_64"]
```

i386 has not been supported for years
```suggestion
  "only-arches": [ "x86_64"]
```

---

what about aarch64?

---

I added `aarch64` but will not be able to test it